lib/takeaway.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'menu'
+require_relative 'order'
+require_relative 'sms'
+
+class Takeaway
+  attr_reader :menu
+
+  def initialize
+    @new_order
+  end
+
+  def print_menu
+    @menu = Menu.new
+    @menu.dishes_list
+  end
+
+  def start_order
+    @new_order = Order.new
+    @new_order.order
+  end
+
+  def add_chicken
+    @new_order = Order.new if @new_order == nil
```

Comment:
It feels like having to check if `@new_order` has been created on `add_chicken`, `add_lamb` and `add_beef` is repetitive.  Could you refactor this so that you only have to write that line once? Perhaps by moving that code to another function? Or even by making sure `@new_order` is always created when Takeaway is instantiated?